types/puppeteer/index.d.ts

```diff
@@ -1331,6 +1327,137 @@ export interface GeoOptions {
 
 export type MediaType = "screen" | "print";
 
+export interface SnapshopOptions {
+/**
```

Comment:
nit: indentation is off here

Reply:
And I'd move this interface down to right above the function declaration where it is used.